examples/psyir/create.py

```diff
@@ -90,52 +90,72 @@ def create_psyir_tree():
 
     # Nodes which do not have Nodes as children and (some) predefined
     # scalar datatypes
-    zero = Literal("0.0", REAL_TYPE)
-    one = Literal("1.0", REAL4_TYPE)
+    # Issue #1136 looks at how to avoid all of the _x versions
```

Comment:
The issue reference should have the TODO tag

Reply:
Added.